### DISCUSSION TITLE
Trame-Vuetify Widgets v3.VExpansionPanel

### DISCUSSION THREAD

**Discussion by ghost** [Kitware/trame#559] [General] [answered]
Hello, I would like to implement v3.VExpansionPanel in my app's UI. I'm not sure about the syntax, but the expansion panel is not showing on the app. I was trying to follow this script as reference from (https://github.com/Kitware/trame-vuetify/blob/master/trame_vuetify/widgets/vuetify3.py):

![image](https://github.com/Kitware/trame/assets/93155074/418912fa-9723-487b-8fcd-200166d386e5)
 Here is my codes:
<img width="1594" alt="Untitled" src="https://github.com/Kitware/trame/assets/93155074/c61f0754-d07f-437f-bb35-f635c0332583">

**Comment by jourdain** [comment]
https://vuetifyjs.com/en/components/expansion-panels/#usage

**Reply by ghost** [reply]
I'm not sure how do I add contents for v3.VExpansionPanel(). Can you please provide a simple example of that?

**Reply by jourdain** [reply, marked as answer]
Doc 

```html
<v-expansion-panels>
  <v-expansion-panel
    title="Title"
    text="Lorem ipsum dolor sit amet consectetur adipisicing elit. Commodi, ratione debitis quis est labore voluptatibus! Eaque cupiditate minima"
  >
  </v-expansion-panel>
</v-expansion-panels>
```

Python

```python
with v3.VExpensionPanels():
   v3.VExpensionPanel(title="Title", text="Hello")
```

**Reply by ghost** [reply]
Got it. Thank you :)

**Reply by ghost** [reply]
Hello, I have a follow up question about the Vuetify Expansion panel. I'm trying to add sliders, buttons, and texts under one expansion panel. I couldn't find any references for this. 

Here is the original Vuetify script: 
![image](https://github.com/Kitware/trame/assets/93155074/b79e2bd5-a850-4b30-8300-a136398bcc61)
Here is the my script in trame:
![image](https://github.com/Kitware/trame/assets/93155074/f6c6a297-807f-4890-acf3-41fda17fe647)
It didn't show anything on the app:
![image](https://github.com/Kitware/trame/assets/93155074/af420fc9-12d5-43eb-9638-1be78f40b8ef)

**Reply by jourdain** [reply]
FYI: your screenshot show the solution.

**Reply by ghost** [reply]
Yeah, I was trying to write the script just like the first screenshot, but it doesn't show anything.

**Reply by jourdain** [reply]
You did not wrote the same thing. Do you expect 2 different code to produce the same result?

You can validate your vue template from trame by printing any of its UI element.

**Reply by ghost** [reply]
It doesn't look same. Can you point out which part of my script is wrong and how do I correct it?

![image](https://github.com/Kitware/trame/assets/93155074/2c79b47f-f808-4c1b-9f3b-0ee050ff4ec2)
![image](https://github.com/Kitware/trame/assets/93155074/15cf3671-a152-4438-8b25-9c342ba10286)

**Reply by jourdain** [reply]
If you want support you can [go here](https://www.kitware.com/trame/).

Otherwise you can look at slide 8 from the [available course](https://kitware.github.io/trame/guide/intro/course.html).

Side note, I don't see where you write in your python `v-slot:default="{ expended }"` on your template.

**Reply by jourdain** [reply]
Please put text in your post not images. Otherwise we can't easily provide you a quick fix from basic typo... Like I can't guide you with the print that you did not understand. Because I can not paste code to be very explicit.
